### ISSUE TITLE
RetroDebugger windows version does not start if the audio device is not available

### ISSUE THREAD

**Issue by zackarya** [slajerek/RetroDebugger#33]
The program does not start if the audio device is unavailable

1. Unplug audio from the computer
2. Launch the program
3. Program does not start
4. No errors are shown

**Desktop (please complete the following information):**
 - OS: Windows
 - Version 10 and 11 (only ones I've tested)

**Comment by unorig** [comment]
@zackarya thanks for raising this. I couldnt figure out why Retro Debugger was not loading. Was driving me crazy.

**Comment by slajerek** [comment]
True, that's known problem. The audio engine drives some parts of emulation, I need to create a fallback virtual audio device and indeed it just now closes with error spit out to console log (thus even not visible on Windows). I'll look at this.

**Comment by slajerek** [comment]
Fixed with 1.0.2
